src/index.html

```diff
@@ -16,17 +29,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="search-panel">
+      <form
+        class="search-form big"
+        data-qa="keypress"
```

Comment:
The `data-qa` attribute is correctly set to `keypress` for the big search form, which aligns with the task requirements.